py/desimodel/io.py

```diff
@@ -131,37 +131,57 @@ def load_tiles(onlydesi=True, extra=False):
         If ``True``, trim to just the tiles in the DESI footprint.
     extra : :class:`bool`, (default False)
         If ``True``, include extra layers with PROGRAM='EXTRA'.
+    tilesfile : (str) name of tiles file to load; or None for default
+        Without path, look in $DESIMODEL/data/footprint, otherwise load file
+    cache : :class:`bool`, (default True)
+        Use cache of tiles data
     """
     global _tiles
-    if _tiles is None:
-        footprint = os.path.join(os.environ['DESIMODEL'],'data','footprint','desi-tiles.fits')
-        with fits.open(footprint) as hdulist:
-            _tiles = hdulist[1].data
+
+    if tilesfile is None:
+        tilesfile = 'desi-tiles.fits'
+
+    tilespath, filename = os.path.split(tilesfile)
+    if tilespath == '':
```

Comment:
use of `os.path.isabs` would be more explicit here ([link](https://docs.python.org/3.5/library/os.path.html#os.path.isabs))